cmd/cbuild/commands/build/buildcprj.go

```diff
@@ -6,12 +6,12 @@
 package build
 
 import (
-	"cbuild/pkg/builder"
-	"cbuild/pkg/builder/cproject"
-	"cbuild/pkg/utils"
 	"errors"
 	"path/filepath"
 
+	"github.com/Open-CMSIS-Pack/cbuild/v2/pkg/builder"
+	"github.com/Open-CMSIS-Pack/cbuild/v2/pkg/builder/cproject"
+	"github.com/Open-CMSIS-Pack/cbuild/v2/pkg/utils"
```

Comment:
All module paths contain `v2` directory. But the directory doesn't exist.
Surprisingly the code builds without any issues. Am I missing some understanding here?

Reply:
Ah I see, In the module’s `go.mod` file, the module path is revised to "github.com/Open-CMSIS-Pack/cbuild/v2"

Reply:
When importing modules with major version v2 and beyond it seems the version needs to be explicitly in the module path (some info [here](https://elliotchance.medium.com/major-versions-in-go-modules-explained-ec7c1df3888b)).

I faced this problem when importing cbuild packages into the new cbuild2cmake tool (WIP PR https://github.com/Open-CMSIS-Pack/cbuild2cmake/pull/6)